dev/ProgressRing/ProgressRing.xaml

```diff
@@ -15,35 +14,62 @@
         <Setter Property="MinWidth" Value="16" />
         <Setter Property="IsTabStop" Value="False" />
         <Setter Property="Width" Value="32" />
+        <Setter Property="Maximum" Value="100" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:ProgressRing">
                     <Grid x:Name="LayoutRoot" Background="Transparent">
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
-                                <VisualState x:Name="Active">
+                                <VisualStateGroup.Transitions>
+                                    <VisualTransition From="Active" To="DeterminateActive">
+                                        <Storyboard>
+                                            <DoubleAnimation
+                                                Storyboard.TargetName="DeterminateLottiePlayer"
+                                                Storyboard.TargetProperty="Opacity"
+                                                To="1"
+                                                Duration="0:0:0.15" />
+                                            <DoubleAnimation
+                                                Storyboard.TargetName="IndeterminateLottiePlayer"
+                                                Storyboard.TargetProperty="Opacity"
+                                                To="0"
+                                                Duration="0:0:0.15" />
```

Comment:
What happens when the Windows Setting for disabling animations is turned on, does this animation get turned off? I'm having trouble remembering.

Reply:
I believe the storyboard will snap instead of animating. @codendone to verify.

Reply:
Yes, VSM storyboards should snap in that case.